### PR TITLE
Allow for extending documents

### PR DESCRIPTION
**Pull request by pamil**

### CODE REVIEW

**Review by Arminek — APPROVED** [review]

File: src/Controller/PriceView.php [hunk 1]
```diff
@@ -9,7 +9,7 @@ class PriceView
     /**
      * @var int
      */
-    public $current;
+    public $amount;
```
Comment (lchrusciel) [inline src/Controller/PriceView.php:12]: ~Why did you change this? It is breaking our API.~

File: src/Factory/ProductListViewFactory.php [hunk 2]
```diff
@@ -162,7 +162,7 @@ private function getPriceView(PriceDocument $price)
     {
         /** @var PriceView $priceView */
         $priceView = new $this->priceViewClass();
-        $priceView->current = $price->getAmount();
+        $priceView->amount = $price->getAmount();
```
Comment (lchrusciel) [inline src/Factory/ProductListViewFactory.php:165]: ~The `current` name has been used with extending price field in mind. In the case, we would like to add an original price how the fields should be named? Anyway, the `PriceView` is also a part of an API of ShopApiPlugin and this change breaks it. Maybe the field name is a little bit unfortunate, but the change should be considered in the scope of whole ShopApiPlugin, not because of custom PriceDocument.~

**Review by lchrusciel — COMMENTED** [review]
~We need to fix this~

**Comment by lchrusciel** [comment]
Nvm, I didn't notice #13
